google-protobuf/google-protobuf-tests.ts

```diff
@@ -1,5 +1,18 @@
 import * as jspb from "google-protobuf";
 
+import "google/protobuf/compiler/plugin_pb";
```

Comment:
this does not seem like the module name the file below defines, the file defines `"google-protobuf/google/protobuf/compiler/plugin_pb"`, is that the intention?